solana-programs/anchor/audius-data/lib/lib.ts

```diff
@@ -269,14 +288,267 @@ export const initUserSolPubkey = async ({
   return provider.send(tx);
 };
 
-/// Create a user without Audius Admin account
+/// Create a content node with the audius admin authority
+type CreateContentNode = {
```

Comment:
nit the types were moved to the top so we can re-use

Reply:
are these reused thought? It not, I think it's easier to co-locate them with the function.
Will change for consistency now though.

Reply:
Some like CreateEntityParams are re-used. Maybe it's better to put them next to each other? 🤷‍♂️